shared/src/hooks/useFetchLiquidityGaugeV5Data.ts

```diff
@@ -32,7 +41,13 @@ const useFetchLiquidityGaugeV5Data = (): LiquidityGaugeV5PoolData => {
   const [, setMulticallCounter] = useState(0);
 
   const doMulticall = useCallback(async () => {
-    if (!minterContract || !gaugeControllerContract) {
+    chainId && console.log(isEthNetwork(chainId));
+    if (
+      // If a wallet is connected and its NOT ETH, dont fetch
+      (chainId && !isEthNetwork(chainId)) ||
```

Comment:
this allows the fetching to continue even when `chainId =undefined`, because it will be using `ethWeb3Context`